cmd/gqlgen/gqlgen.go

```diff
@@ -1,10 +0,0 @@
-//go:build tools
```

Comment:
This file isn't needed anymore as of Go 1.24 because it introduced [better tool dependency management](https://tip.golang.org/doc/go1.24#tools). The tool version and dependency is managed in `go.mod`.